internal/pubsub/example_test.go

```diff
@@ -78,8 +79,11 @@ func Example_sendReceiveMultipleMessages() {
 
 	// Receive messages from the subscription.
 	ms2 := []string{}
```

Comment:
Can we use
```
var ms2 []string
```
instead? It's a bit more idiomatic.

Reply:
Done.